internal/gatewayapi/translator_test.go

```diff
@@ -28,7 +28,7 @@ func mustUnmarshal(t *testing.T, val string, out interface{}) {
 }
 
 func TestTranslate(t *testing.T) {
-	inputFiles, err := filepath.Glob(filepath.Join("testdata", "*.in.yaml"))
+	inputFiles, err := filepath.Glob(filepath.Join("testdata", ".in.yaml"))
```

Comment:
I believe with this change, all tests are no longer running.
Here's the coverage (5%) from the package in CI
https://github.com/envoyproxy/gateway/actions/runs/3635220345/jobs/6134115423#step:4:23

and here is the coverage (77%) when I run in on the main branch
```
ok  	github.com/envoyproxy/gateway/internal/gatewayapi	1.697s	coverage: 77.3% of statements
```

Reply:
np, coverage looks good now